routes/index.js

```diff
@@ -0,0 +1,48 @@
+var express = require('express');
+var router = express.Router();
+
+//GET RANDOM COLOR
+function randomColor () {
+  var colors = ["#FFBF00", "#0080FF", "#01DF3A", "#FF0080"];
+  return colors[Math.floor(Math.random() * colors.length)];
+}
+
+//GET RANDOM COMPLIMENTS
+function getRandomCompliments() {
```

Comment:
This function is named `getRandomCompliments` but returns a single compliment. I'd name it `getRandomCompliment` for consistency